pipelines/google_sheets/data_validator.py

```diff
@@ -25,13 +27,10 @@ def get_data_type(value_list: list[DictStrAny]) -> list[bool]:
     value_type_list = []
     # loop through the list and process each value dict, decide if something is a datetime value or not
     for val_dict in value_list:
-        # TODO: try except
-        is_date = "effectiveValue" in val_dict and \
-                  "numberValue" in val_dict['effectiveValue'] and \
-                  "effectiveFormat" in val_dict and \
-                  "numberFormat" in val_dict['effectiveFormat'] and\
-                  "type" in val_dict["effectiveFormat"]["numberFormat"] and\
-                  ("DATE" in val_dict["effectiveFormat"]["numberFormat"]["type"] or "TIME" in val_dict["effectiveFormat"]["numberFormat"]["type"])
+        try:
+            is_date = "DATE" in val_dict["effectiveFormat"]["numberFormat"]["type"] or "TIME" in val_dict["effectiveFormat"]["numberFormat"]["type"]
```

Comment:
why are you using `in`? "type" is a string. you could just compare, otherwise it is looking for substrings. is this your intention?